librb/src/openssl.c

```diff
@@ -413,13 +423,41 @@ rb_setup_ssl_server(const char *const certfile, const char *keyfile,
 		}
 	}
 
-	if(SSL_CTX_set_cipher_list(ssl_ctx_new, cipherlist) != 1)
+
+	int ret1 = SSL_CTX_set_cipher_list(ssl_ctx_new, cipherlist_old);
```

Comment:
`ret_old` and `ret_new` might make this bit easier to follow

Reply:
Done.